portsyncd/portsyncd.cpp

```diff
@@ -69,7 +71,7 @@ int main(int argc, char **argv)
     DBConnector cfgDb(CONFIG_DB, DBConnector::DEFAULT_UNIXSOCKET, 0);
     DBConnector appl_db(APPL_DB, DBConnector::DEFAULT_UNIXSOCKET, 0);
     DBConnector state_db(STATE_DB, DBConnector::DEFAULT_UNIXSOCKET, 0);
-    ProducerStateTable p(&appl_db, APP_PORT_TABLE_NAME);
```

Comment:
why change to producer state table?

Reply:
Actually change to ProducerTable. The entries in APP_PORT_TABLE_NAME are sequence sensitive because there are event entries.


---
In reply to: [205855555](https://github.com/Azure/sonic-swss/pull/551#discussion_r205855555) [](ancestors = 205855555)

Reply:
what event entries?